en/asgardeo/docs/apis/organization-apis/restapis/branding-management.yaml

```diff
@@ -463,6 +464,18 @@ components:
       schema:
         type: string
       example: "login"
+    restrictToPublishedQueryParam:
+      in: query
+      name: restrictToPublished
+      required: false
+      description: |
+        Specifies whether to use only published branding preferences for resolving.
```

Comment:
```suggestion
        Specifies whether to limit resolving to published branding preferences.
```

Reply:
Addressed with https://github.com/wso2/docs-is/pull/4701/commits/169604e8cfb41500a9204a3d166a7c647c390c98